src/lib/SSVKeys.ts

```diff
@@ -20,56 +20,32 @@ export interface IPayloadMetaData {
  */
 export class SSVKeys {
   static SHARES_FORMAT_ABI = 'abi';
-  static VERSION = {
-    V2: 'v2',
-    V3: 'v3',
-  };
 
-  protected version: string;
   protected threshold: ISharesKeyPairs | undefined;
 
-  public keySharesInstance: KeyShares;
-  public privateKey: any;
-  public publicKey: any;
-
-  constructor(ver: string) {
-    if (!Object.values(SSVKeys.VERSION).includes(ver)) {
-      throw Error ('Version is not supported');
-    }
-
-    this.version = ver;
-    this.keySharesInstance = new KeyShares({ version: this.version });
-  }
-
-  get keyShares(): KeyShares {
-    return this.keySharesInstance;
-  }
   /**
    * Extract private key from keystore data using keystore password.
    * Generally can be used in browsers when the keystore data has been provided by browser.
    * @param data
    * @param password
    */
-  async getPrivateKeyFromKeystoreData(data: string, password: string): Promise<string> {
-    try {
-      const privateKey = await new EthereumKeyStore(data).getPrivateKey(password);
-
-      await bls.init(bls.BLS12_381);
-      this.privateKey = `0x${bls.deserializeHexStrToSecretKey(privateKey).serializeToHexStr()}`;
-      this.publicKey = `0x${bls.deserializeHexStrToSecretKey(privateKey).getPublicKey().serializeToHexStr()}`;
-      return privateKey;
-    } catch (error: any) {
-      return error;
-    }
+  async extractKeys(data: string, password: string): Promise<ExtractedKeys> {
+    const privateKey = await new EthereumKeyStore(data).getPrivateKey(password);
+    await bls.init(bls.BLS12_381);
```

Comment:
I look inside of bls.init method and it's heavy. If we need parallel workers to be fast we should init it only once. Please implement this

Reply:
@vadiminc I don't see it is initializing only once, did you commit it?

Reply:
If you take a look here:
https://github.com/bloxapp/ssv-keys/pull/37/files#diff-c127a9a3b0b9b65b6bb86b567f9cd4013f3f8b0bb89019301c2846b3882e7157L56 

extractKeys is called inside of expressjs route, it means that each time you run this endpoint in parallel nature it will run bls.init, and inside of it it will do that heavy work each time, even more - with the same instance of bls object.

Regarding the fact that ssv-keys strictly works with only one BLS type (12_381), to initialize it only once and bring simple check:

```
if (!this.initialized) bls.init()
```